src/SUMMARY.md

```diff
@@ -1,56 +1,61 @@
 # Summary
 
-[Introduction](./README.md)
+[Home](./README.md)
 
-# Getting Started
+- [Introduction](./ch00-00-introduction.md)
 
 - [Getting Started](./getting-started/getting-started.md)
-  - [Installation](./getting-started/installation.md)
-  - [Choosing a Backend](./getting-started/choosing-a-backend.md)
 
-# Tutorial
+  - [Installation](./getting-started/installation.md)
+  - [Features](./getting-started/features.md)
+  - [Choosing a Backend]()
 
 - [Tutorial](./tutorial/README.md)
+
   - [App.rs - Holding application state](./tutorial/app.md)
   - [Main.rs - UI loop and event handling](./tutorial/main.md)
   - [Ui.rs - Layouts, widgets, frames, oh my!](./tutorial/ui.md)
   - [Conclusion](./tutorial/closing_thoughts.md)
 
-# How To
-
 - [How To]()
-  - [Blocks](./how-to/block.md)
-  - [Displaying Text](./how-to/displaying-text.md)
-    - [Styling-Text](./how-to/styling-text.md)
-    - [Paragraphs](./how-to/paragraph.md)
+
+  - [Blocks]()
+  - [Displaying Text]()
+    - [Styling-Text]()
+    - [Paragraphs]()
   - [Layouts]()
-    - [Layout Constraints](./how-to/layout-constraints.md)
+    - [Layout Constraints]()
   - [Layout Widgets]()
   - [Create a Widget]()
   - [Handle Panics]()
 
-# References
+- [References]()
 
-- [Crate Docs]()
-- [Starter Template]()
-- [Third Party Crates]()
+  - [Crate Docs]()
+  - [Starter Template]()
+  - [Third Party Crates]()
 
-# Concepts
+- [Concepts]()
 
-- [Rendering](./concepts/rendering.md)
-- [Event Handling](./concepts/event_handling.md)
-- [Best Practices]()
+  - [Rendering]()
+  - [Event Handling](./concepts/event_handling.md)
+  - [Best Practices]()
   - [Project Structure]()
-- [Application Patterns]()
+  - [Application Patterns]()
   - [Design Patterns]()
   - [Application State]()
   - [Event Handling]()
   - [Key Binding]()
-- [Threading]()
-- [Logging]()
-- [Configuration]()
-- [Backend Choice]()
-- [Other stuff]()
+  - [Threading]()
+  - [Logging]()
+  - [Configuration]()
+  - [Backend Choice]()
+  - [FAQ](./concepts/faq.md)
```

Comment:
This needs to be "above the fold" (i.e. higher in the list so that it renders when the user doesn't scroll the TOC on a reasonable screen size - perhaps the first item under concepts? Also we might need to enable folding: https://rust-lang.github.io/mdBook/format/configuration/renderers.html#outputhtmlfold

I also moved the tutorial steps under a name that describes what will be built.

<img width="311" alt="image" src="https://github.com/ratatui-org/ratatui-book/assets/381361/d3757631-8901-4646-817a-143675174ad6">
vs:
<img width="301" alt="image" src="https://github.com/ratatui-org/ratatui-book/assets/381361/8ed03075-6896-4c84-adb8-ec780231cfbe">

Reply:
I agree. I debated between putting `FAQ` at the top of `Concepts`.

I actually think `FAQ` should be up one up more level in organization. As we add more faq content, I can imagine it not fitting in with "concepts". Also, skimming through FAQ is enough for most people to know it exists and return to it if they need a solution discussed there.

Reply:
Perhaps before References then?

Reply:
Resolved in https://github.com/ratatui-org/ratatui-book/pull/15/commits/fd62fbe78ff9899f04bc12c8cf85fc15259d7ab9